salt/elasticsearch/files/ingest/zeek.opcua_binary

```diff
@@ -14,6 +14,8 @@
     { "rename":   { "field": "message2.request_id",			"target_field": "opcua.request_id",			"ignore_missing": true } },
     { "rename":   { "field": "message2.namespace_idx",			"target_field": "opcua.namespace_index",		"ignore_missing": true } },
     { "rename":   { "field": "message2.encoding_mask",			"target_field": "opcua.encoding_mask",			"ignore_missing": true } },
+    { "convert":  { "field": "opcua.encoding_mask", 			"type": "string",
+ "ignore_missing": true }},
```

Comment:
Please add a space between the two last curly braces on this line to match the other lines.

Reply:
(and do the same for the other new convert lines)